core/spec/models/spree/calculator/default_tax_spec.rb

```diff
@@ -9,6 +10,11 @@
   let(:included_in_price) { false }
   subject(:calculator) { Spree::Calculator::DefaultTax.new(calculable: rate ) }
 
+  describe ".description" do
```

Comment:
You can even put the whole `describe` block including the `subject` in the shared example so that you only have to leave `it_behaves_like` in the calculator specs.